### PR TITLE
Badge app for reset team members

### PR DESCRIPTION
**Pull request by buoyad**
Adds badges to the dock icon (on desktop), teams tab, team row, and members tab.

Still needs work on a badge dismissal strategy (cc @patrickxb)

@keybase/react-hackers this is ready!

### CODE REVIEW

**Comment by akalin-keybase** [comment]
Sorry I didn't get to this! Mind rebasing it first?

**Comment by buoyad** [comment]
@akalin-keybase sure, ptal!

File: shared/teams/main/team-list.js [hunk 1]
```diff
@@ -114,6 +123,7 @@ const TeamList = (props: Props) => (
         onOpenFolder={() => props.onOpenFolder(name)}
         onManageChat={() => props.onManageChat(name)}
         onViewTeam={() => props.onViewTeam(name)}
+        resetUserCount={(props.teamresetusers[name] && props.teamresetusers[name].size) || 0}
```
Comment (akalin-keybase) [inline shared/teams/main/team-list.js]: a ternary here instead of && and || seems more natural

File: shared/teams/team/container.js [hunk 2]
```diff
@@ -60,6 +61,7 @@ const mapStateToProps = (state: TypedState, {routeProps, routeState}): StateProp
     you: state.config.username,
     yourRole: Constants.getRole(state, teamname),
     yourOperations: Constants.getCanPerform(state, teamname),
+    resetUserCount: state.entities.getIn(['teams', 'teamNameToResetUsers', teamname], I.Set()).size,
```
Comment (akalin-keybase) [inline shared/teams/team/container.js]: `getIn([..., teamname, 'size'], 0)` better?

Comment (akalin-keybase) [reply shared/teams/team/container.js]: nevermind!

**Pull request merged**